integration-tests/tests/test_integration.py

```diff
@@ -532,11 +623,16 @@ async def test_publish_forever(self, client: PythAgentClient):
         # Get the price account with which to send updates
         price_account = product["price_accounts"][0]["account"]
 
+        product_unperm = products[ETH_USD["attr_dict"]["symbol"]]
+        product_account_unperm = product_unperm["account"]
+        # Get the price account with which to send updates
+        price_account_unperm = product_unperm["price_accounts"][0]["account"]
+
         while True:
             # Send an "update_price" request
             await client.update_price(price_account, 47, 2, "trading")
             time.sleep(1)
 
             # Send another "update_price" request to trigger aggregation
-            await client.update_price(price_account, 81, 1, "trading")
+            await client.update_price(price_account_unperm, 81, 1, "trading")
```

Comment:
do you think this is the right behavior for the infinite looping test? I haven't thought through what's most useful for this test case, and i could see the warnings being annoying. but maybe there's a good reason to have the unpermissioned symbols here too.

Reply:
I don't think it hurts to have it, but neither does just valid publishing, removed.